src/main/resources/assets/numismatic-overhaul/lang/uk_ua.json

```diff
@@ -0,0 +1,69 @@
+{
+  "item.numismatic-overhaul.bronze_coin": "Бронзова монета",
+  "item.numismatic-overhaul.silver_coin": "Срібна монета",
+  "item.numismatic-overhaul.gold_coin": "Золота монета",
+  "item.numismatic-overhaul.money_bag": "Мішок із грішми",
+  "block.numismatic-overhaul.shop": "Крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop": "Невичерпна крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop.tooltip": "Лише в творчості — ніколи не вичерпуються запаси",
```

Comment:
does this mean "Creative (mode) only", or "Creativity only"? the en_us translation is about creative mode

Reply:
The exact translation will be "Only in creative"